internal/data/receivers_wallet.go

```diff
@@ -616,3 +576,115 @@ func (rw *ReceiverWalletModel) UpdateInvitationSentAt(ctx context.Context, sqlEx
 
 	return receiverWallets, nil
 }
+
+type ReceiverWalletUpdate struct {
+	Status                      *ReceiversWalletStatus `db:"status"`
+	AnchorPlatformTransactionID *string                `db:"anchor_platform_transaction_id"`
+	StellarAddress              *string                `db:"stellar_address"`
+	StellarMemo                 *string                `db:"stellar_memo"`
+	StellarMemoType             *string                `db:"stellar_memo_type"`
+	OTPConfirmedAt              *time.Time             `db:"otp_confirmed_at"`
+	OTPConfirmedWith            *string                `db:"otp_confirmed_with"`
+}
```

Comment:
Do we need these fields to be pointers?

Reply:
Addressed in c174a217289def231177fad9bcfc56475f2200e7